bikespace_frontend/src/components/FeedbackMailTo.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {StaticImage} from 'gatsby-plugin-image';
+
+const feedback_to = 'bikespaceto@gmail.com';
+const feedback_subject = 'BikeSpace App Feedback';
+const feedback_body = `----------
```

Comment:
Could we follow the google typescript guide for camelcase for variable declarations. The linter should've picked these up 🤔 
https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type